concepticondata/concepticon.tsv

```diff
@@ -3980,3 +3980,14 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3994	PIANO (INSTRUMENT)	Speech and language	A large musical instrument with a row of black and white keys that are pressed to play notes.	Person/Thing
 3995	AMAZONIAN MANATEE	Animals	Trichechus inunguis ([GBF ID: 2435292](https://www.gbif.org/species/2435292)), one of the four living Manatee species. Native to the Amazon river system, the amazonian manatee is a fully aquatic mammal.	Person/Thing
 3996	TOP (TOY)	The physical world	A toy with rounded sides, a flat top, a vertical handle and a point at the bottom that turns round and round on the point when the handle is pushed.	Person/Thing	
+3997	ARTICHOKE	Agriculture and vegetation	A plant with a round mass of pointed parts like leaves surrounding its flower that are eaten as a vegetable.	Person/Thing
+3998	IRONING BOARD	Modern world	A board covered with cloth, usually with folding legs, on which clothes and other items can be made smooth with an iron.	Person/Thing
+3999	ROCKING CHAIR	The house	A chair built on two pieces of curved wood so that it moves forwards and backwards when you are sitting in it.	Person/Thing	
```

Comment:
It might be good to add a relation to the broader concept chair in the `conceptrelations.tsv`.

Reply:
done! :-)